README.md

```diff
@@ -1,21 +1,21 @@
 # Insight UI
 
-A Dash blockchain explorer web application service for [Dashcore Node](https://github.com/dashevo/dashcore-node) using [Insight API](https://github.com/dashevo/insight-api).
+A Dash blockchain explorer web application service for [Dashcore Node](https://github.com/dashpay/dashcore-node) using [Insight API](https://github.com/dashpay/insight-api).
 
 ## Quick Start
 
-Please see the guide at [https://bitcore.io/guides/full-node](https://bitcore.io/guides/full-node) for information about getting a block explorer running. This is only the front-end component of the block explorer, and is packaged together with all of the necessary components in [Dashcore](https://github.com/dashevo/dashcore).
+Please see the guide at [https://bitcore.io/guides/full-node](https://bitcore.io/guides/full-node) for information about getting a block explorer running. This is only the front-end component of the block explorer, and is packaged together with all of the necessary components in [Dashcore](https://github.com/dashpay/dash).
 
 ## Getting Started
 
 To manually install all of the necessary components, you can run these commands:
 
 ```bash
-npm install -g @dashevo/dashcore-node
+npm install -g @dashpay/dashcore-node
```

Comment:
Its still @dashevo in the npm packages, so I'm not sure it will work

https://www.npmjs.com/package/@dashevo/insight-ui

Reply:
Reverted